awpy/plot/plot.py

```diff
@@ -40,6 +46,10 @@ def plot(  # noqa: PLR0915
             - 'armor': int (0-100)
             - 'direction': Tuple[float, float] (pitch, yaw in degrees)
             - 'label': str (optional)
+        ignore_extreme_points (bool, optional): If set to True, will ignore
```

Comment:
This is an interesting proposition. Is there any reason we would want to allow this in the first place? Or, in other words, do you think there's any danger to making this a default?

Reply:
If I were designing this function from the ground-up, I would have the default set to `True`.

In the commit I have set it to default as `False` because it mimics the previous behavior of the function (i.e. it plotted points way outside of the map). My reasoning was backwards compatibility for people who have already written code with the current behavior in mind.

I'm kind of new to GitHub and commits so I'm leaving the decision up to you :D

Reply:
Totally understand the backwards compatibility concerns. I think given that this is still a beta version, we can feel ok with breaking the previous functionality, particularly in this case, where there is little noticeable value to plotting points outside the map. I say we remove this arg and make it default behavior! It's a nice safeguard that I overlooked when first writing the function.

Reply:
Sounds good, will remove the arg and integrate the functionality.